README.md

```diff
@@ -26,16 +26,12 @@ My first project at Microverse
 
 # 📖 [Hello-microverse-project] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
-
 **[Hello-microverse-project]** is my First Project at Microverse
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
```

Comment:
- [ ] In the Tech stack section of the README, kindly remove the technologies listed in the `Client`, `Server` and `Database` subsections. Those sections contain technologies that were not used in this project. Kindly remove the `Server` and `Database` subsections and replace those in the `Client` subsections with HTML and CSS. 👍 😎